src/pull-retry.ts

```diff
@@ -19,15 +19,11 @@ import {StatusObject, status} from '@grpc/grpc-js';
  * retryable status codes
  */
 export const RETRY_CODES: status[] = [
-  status.OK,
```

Comment:
For TypeScript users, is this technically a breaking change?  Not sure how/if this is exported from `index`.

Reply:
It's a list of what statues might trigger a retry, which got pared down. It's honestly a bit odd that it's even exported. (I removed the export and reran the tests, and they're happy still... but I don't want to do that for fear of it being a real breaking change in case someone's checking it in their own client code.)